dtmi/meshsystems/ble/publisher-1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@id": "dtmi:MeshSystems:ble:publisher;1",
+  "@type": "Interface",
+  "@context": "dtmi:dtdl:context;2",
+  "displayName": "BLE Publisher",
+  "contents": [
+    {
+      "@type": "Property",
+      "writable": true,
+      "displayName": {
+        "en": "ble Advertisement"
+      },
+      "name": "bleAdvertisement",
+      "description": "Publisher Advertisement",
+      "schema": {
+        "@type": "Object",
+        "fields": [
+          {
+            "name": "raw",
+            "displayName": "Raw Advertisement",
+            "schema": "string"
+          },
+          {
+            "name": "interval",
+            "displayName": "Interval",
+            "schema": "string"
+          },
+          {
+            "name": "phy",
+            "displayName": "PHY",
+            "schema": "string"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Property",
+      "displayName": {
+        "en": "bleFwVersion"
+      },
+      "name": "bleFwVersion",
+      "description": "Bluetooth module's firmware version",
```

Comment:
Do you want to include punctuation, like a period . at the end of the descriptions of this model set  (as described in the naming guideline best practices https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) ?

Reply:
followed the best practices as recommended